WordPressComKitTests/MeServiceTests.swift

```diff
@@ -0,0 +1,55 @@
+import XCTest
+import OHHTTPStubs
+import WordPressComKit
+
+class MeServiceTests: XCTestCase {
+    var subject: MeService!
+    
+    override func setUp() {
+        super.setUp()
+        
+        subject = MeService()
+    }
+    
+    override func tearDown() {
+        super.tearDown()
+        
+        subject = nil
+        OHHTTPStubs.removeAllStubs()
+    }
+    
+    func testFetchMe() {
+        stub(isMethodGET() && isHost("public-api.wordpress.com") && isPath("/rest/v1.1/me")) { _ in
```

Comment:
Totally agreed that mappers need to be tested separate from the services. [Logged here](https://github.com/Automattic/WordPressComKit/issues/6).

I definitely want to test more scenarios like various HTTP errors and such. If we get some of that error handling boiler plate into a central place it can be unit tested there. For example if the service call is interrupted by an interstitial page (like at a coffee shop when asking for the user to input a code or click accept) then I want to make sure the service doesn't crash.